alicloud/table_alicloud_rds_backup.go

```diff
@@ -0,0 +1,275 @@
+package alicloud
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/rds"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudRdsBackup(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_rds_backup",
+		Description: "ApsaraDB RDS Backup is a policy expression that defines when and how you want to back up your DB Instances.",
+		List: &plugin.ListConfig{
+			ParentHydrate: listRdsInstances,
+			Hydrate:       listRdsBackups,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "backup_id", Require: plugin.Optional},
+				{Name: "db_instance_id", Require: plugin.Optional},
+				{Name: "backup_status", Require: plugin.Optional},
+				{Name: "backup_mode", Require: plugin.Optional},
+				{Name: "backup_start_time", Require: plugin.Optional, Operators: []string{"="}},
+				{Name: "backup_end_time", Require: plugin.Optional, Operators: []string{"="}},
+				{Name: "backup_location", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: []*plugin.Column{
+			// Top columns
+			{
+				Name:        "backup_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the backup set.",
+			},
+			{
+				Name:        "db_instance_id",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBInstanceId"),
+				Description: "The ID of the single instance to query.",
+			},
+			{
+				Name:        "backup_status",
+				Description: "The status of the backup. Valid values: Success, Failed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "backup_mode",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup mode.",
+			},
+			{
+				Name:        "backup_db_names",
```

Comment:
```suggestion
				Name:        "backup_db_name",
```

Should we update it to `backup_db_name` since it is a string and we don't pluralize string columns?